docs/extras/advanced-variables/variable-defaults.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Variable Defaults
+sidebar_position: 2
+---
+
+Variable defaults are the values you provide in code which tell DevCycle SDKs what value to assign to a Variable in
+cases where a different value is not available. 
```

Comment:
instead of "where a different value is not available". I think this should be explicit about a value from the "server" not being available. I know we don't want to suggest there's a call to our central server so we'll need to be careful with wording.

Here's an option:
"in cases where a value you DevCycle remote configuration is not available."

Reply:
that way of phrasing it isn't accurate though, it will still default in cases where the configuration is available if it doesn't include the variable